homeassistant/components/brottsplatskartan/const.py

```diff
@@ -0,0 +1,39 @@
+"""Adds constants for brottsplatskartan integration."""
```

Comment:
I usually recommend that constants are centralized as a preliminary/standalone PR.